docs/config.ini

```diff
@@ -30,7 +34,6 @@ guard=onebot.plugins.acl.user_based_policy
 identify_by = mask
 
 [onebot.plugins.acl]
-# Pre-seed acl
```

Comment:
Oh, okay I understand what you mean. It make sense now. I'll change it back and let you clarify if you want.